web/packages/teleterm/src/env.ts

```diff
@@ -0,0 +1,24 @@
+interface Env {
+  prehogAddress: string;
+  feedbackAddress: string;
+}
+
+let ENV: Env;
+
+// process.env.NODE_ENV is resolved by webpack at compile time
+if (process.env.NODE_ENV === 'development') {
```

Comment:
I double checked that and I can confirm that it works correctly for both the dev and the packaged version of the app. 👍